3rdparty/udis86/libudis86/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+# UJIT: CMake file for udis86 to avoid dependencies on autoconf and libtool.
```

Comment:
In my opinion, this whole file is not necessary and can just be included in udis86/CMakeLists.txt

Reply:
Fixed.